security/aporeto/build/ansible/templates/external_networks/apb-nfs.yml.j2

```diff
@@ -0,0 +1,14 @@
+APIVersion: 1
+label: apb-nfs
+data:
+  externalnetworks:
+  - name: apb-nfs
+    propagate: true
+    entries:
+    - {{ apb_external_nfs_host }}
+    ports:
+    - "22"
+    protocols:
+    - tcp
+    associatedTags:
+    - ext:network=nfs
```

Comment:
Worlds tiniest nit-pick, but should have a newline at end of file. Thanks GitHub for the nice little red indicator!